front/src/applications/editor/components/InfraErrors/InfraErrorCorrectorModal.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { FC, useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useTranslation } from 'react-i18next';
+
+import { ConfirmModal, useModal } from 'common/BootstrapSNCF/ModalSNCF';
+import { Operation, osrdEditoastApi } from 'common/api/osrdEditoastApi';
+import Loader from 'common/Loader';
+import { saveOperations } from 'reducers/editor';
+import { getInfraID } from '../../../../reducers/osrdconf/selectors';
```

Comment:
This might also fix the warning
```suggestion
import { getInfraID } from 'reducers/osrdconf/selectors';
```